feature, source ATN-USDC, NTN-USDC, USDC-USD ratio for the protocol. (#30)
autonity/autonity-oracle: README.md

```yaml
# The forex data plugins are used to fetch realtime rate of currency pairs:
# EUR-USD, JPY-USD, GBP-USD, AUD-USD, CAD-USD and SEK-USD from commercial data providers. There are 4 implemented forex
# data plugins, each of them requires the end user to apply for their own service key from the selected data provider.
# The selection of which forex data plugin(s) to use is for the end user to decide. The user can use any one of them,
# or he/she can use multiple forex data plugins in the setup.
#
# The crypto data plugins are used to fetch realtime rate of crypto currency pairs: ATN-USDC, NTN-USDC, NTN-ATN and

> # USDC-USD. The ATN-USDC, NTN-USDC prices are bridged from Polygon network via a middle data service which provides

Reviewer: ```suggestion
# USDC-USD. USDC liquidity is bridged to the Autonity public testnet from the Polygon Amoy testnet via a bridge service. USDC is used as the quote currency for ATN-USDC and NTN-USDC markets which provides
```

> # open and free data access for the pcgc_cax plugin, the NTN-ATN price is computed by pcgc_cax plugin from the price

Reviewer: ```suggestion
# open and free data access for the `testnet_dax plugin`, the NTN-ATN price is computed by `testnet_dax plugin` from the price
```

> # of ATN-USDC and NTN-USDC. So put the pcgc_cax plugin in your plugin directory, thus oracle server can discovery and

Reviewer: ```suggestion
# of ATN-USDC and NTN-USDC. To retrieve ATN and NTN prices, put the `testnet_dax plugin` in your plugin directory. Oracle server can then discover and
```

> # load it, otherwise your oracle node will be slashed. You don't need config the pcgc_cax plugin, as it use open and

Reviewer: ```suggestion
# load it. Configuring the `testnet_dax` plugin does not require an API key; it is an open and
```

# free data source.

> # More over that, USDC-USD prices are required by the protocol to convert the ATN-USDC and NTN-USDC to ATN-USD and

Reviewer: ```suggestion
# USDC-USD prices are required by the protocol to convert the ATN-USDC and NTN-USDC to ATN-USD and
```

> # NTN-USD. Thus, we implemented 3 different plugins to source the datapoint from open and free data sources: coinbase,

Reviewer: ```suggestion
# NTN-USD. This enables the reporting of ATN and NTN prices in USD to the ASM. To enable this, 3 plugins are implemented to source the USDC-USD datapoint from open and free data sources: coinbase,
```

> # coingecko, and kraken. To prevent single data source failure, we recommend to put all the 3 usdc plugins into your

Reviewer: ```suggestion
# coingecko, and kraken. To prevent single data source failure, putting all 3 usdc plugins into your
```

> # plugin directory, thus oracle server can discovery and load them, otherwise your oracle node might be slashed due to

Reviewer: ```suggestion
# plugin directory is recommended. Oracle server can then discover and load them.
```

> # missing of data report. You don't need to config the usdc plugins as well.

Reviewer: ```suggestion
# You don't need to configure the usdc plugins in your oracle server plugin configuration file.
```

Note: removing the reference to slashing pending slashing implemtnation.  I.e.

># Note that if your oracle server does not submit valid price reports it is at risk of slashing.

#	DataUpdateInterval int    `json:"refresh" yaml:"refresh"`   // the interval in seconds to fetch data due to the rate limit from the provider.
#}

# As an example, to set the configuration of the plugin `forex_currencyfreaks`, only the required field are needed
# however you can configure the optional fields on demand to fit your service quality from the rate provider.
#  - name: forex_currencyfreaks              # required, it is the plugin file name in the plugin directory.
#    key: 175aab9e47e54790bf6d502c48407c10   # required, visit https://currencyfreaks.com to get your key, and replace it.
#    scheme: https                           # optional, https or http, default value is https.
#    endpoint: api.currencyfreaks.com        # optional, default value is api.currencyfreaks.com
#    timeout: 10                             # optional, default value is 10.
#    refresh: 3600                           # optional, default value is 30, that is 30s to fetch data from data source.

> # Un-comment below lines to enable your forex data plugin's configuration on demand. Your production configurations start from below:

Reviewer: ```suggestion
# Un-comment below lines to enable your forex data plugin's configuration on demand. Your production configurations starts from below:
```
